src/debian/iot/arm32v7/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:latest
+FROM debian:bullseye
```

Comment:
Since this is now targeting a specific version, I think it would make sense to move the Dockerfile to the version-specific directory: src/Debian/11/iot/arm32v7/Dockerfile. The manifest.json would need to reflect that path change as well.

Reply:
Done